storage/object/troubleshooting/lost-bucket-access-bucket-policy.mdx

```diff
@@ -0,0 +1,49 @@
+---
+meta:
+  title: I lost access to a bucket after putting a bucket policy
```

Comment:
Instinctively "putting" sounds like the wrong word (should be "setting" or "applying" ?) unless it's a technical thing because you use a PUT action to set the bucket policy?